server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -121,6 +124,11 @@ public ShuffleTaskManager(
     leakShuffleDataCheckExecutorService.scheduleAtFixedRate(
         () -> checkLeakShuffleData(), leakShuffleDataCheckInterval,
             leakShuffleDataCheckInterval, TimeUnit.MILLISECONDS);
+    this.triggerFlushExecutorService = Executors.newSingleThreadScheduledExecutor(
+            ThreadUtils.getThreadFactory("triggerShuffleBufferManagerFlush"));
+    triggerFlushExecutorService.scheduleAtFixedRate(
```

Comment:
Why we need this?

Reply:
Is it only useful for tests?

Reply:
The method maybe help for you
https://github.com/apache/incubator-uniffle/blob/4bd3af64d9e8460301bf04f4b0db261482f63222/internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java#L354